Dockerfile

```diff
@@ -1,30 +1,28 @@
-FROM registry.fedoraproject.org/fedora:38-x86_64
+FROM registry.fedoraproject.org/fedora:39-x86_64
 
 LABEL \
     name="Freshmaker application" \
     vendor="Freshmaker developers" \
-    license="GPLv2+" 
-
+    license="GPLv2+"
 
 # Use Copr repo for python3-rhmsg package
-RUN dnf install -y 'dnf-command(copr)' && dnf copr enable -y qwan/python-rhmsg
+RUN dnf install -y 'dnf-command(copr)' && dnf copr enable -y qwan/python-rhmsg && dnf copr enable -y mfoganho/python3-qpid-proton
 
 COPY yum-packages.txt /tmp/yum-packages.txt
 
 RUN \
-    dnf -y install $(cat /tmp/yum-packages.txt) python3-rhmsg findutils && \
+    dnf -y install $(cat /tmp/yum-packages.txt) && \
     dnf clean all
 
 WORKDIR /src
 
 COPY . .
 
 RUN \
-    # All dependencies should've been installed from RPMs
-    echo '' > requirements.txt && \
+    pip3 install -r requirements.txt && \
     pip3 install . --no-deps
```

Comment:
Is `--no-deps` still needed? Since we're using pip installs the dependencies now, I think we don't need `--no-deps` now, could you check this?

Reply:
It worked! Thanks for spotting that!